.github/dependabot.yaml

```diff
@@ -10,7 +10,7 @@ updates:
       - "openkcm/go-maintainers"
     schedule:
       interval: daily
-    open-pull-requests-limit: 5
+    open-pull-requests-limit: 1
```

Comment:
I don't think that this will actually group updates together. It only limits the number of PRs we get. Maybe have a look on https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/optimizing-pr-creation-version-updates#prioritizing-meaningful-updates. I think this is what we are looking for. I especially like example 3 to group patches and minor updates but have individual updates for major updates as they come with breaking changes.